### ISSUE TITLE
Exception override

### ISSUE THREAD

**Issue by pavelloz** [uBlockOrigin/uBlock-issues#743]
Im on a page - dashboard.render.com
It has segment tracking, and most of its requests are blocked by easylist.
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/546845/66268418-77149f00-e83d-11e9-932c-ba66c59a2d04.png">

But the one in question is in some kind of exception list on easylist (i dont know why anyone would do that, but thats a list issue):
<img width="721" alt="image" src="https://user-images.githubusercontent.com/546845/66268487-1c2f7780-e83e-11e9-81a2-4fd1963b4aa8.png">

Even though i created a filter specifically for the whole domain

> ||cdn.segment.com^$third-party

I think ALWAYS user filters should take precedence over loaded lists filters.

It does override only after i use important directive, which is something i should not do on my machine.

> ||cdn.segment.com^$important,third-party

**Comment by gwarser** [comment]
Duplicate of https://github.com/gorhill/uBlock/issues/1520

See also https://github.com/gorhill/uBlock/issues/3353
